barretenberg/cpp/src/barretenberg/dsl/acir_format/honk_recursion_constraint.cpp

```diff
@@ -208,11 +209,12 @@ PairingPointAccumulatorIndices create_honk_recursion_constraints(
     RecursiveVerifier verifier(&builder, vkey);
     aggregation_state_ct input_agg_obj = bb::stdlib::recursion::convert_witness_indices_to_agg_obj<Builder, bn254>(
         builder, input_aggregation_object_indices);
-    aggregation_state_ct output_agg_object = verifier.verify_proof(proof_fields, input_agg_obj);
+    bb::stdlib::recursion::honk::UltraRecursiveVerifierOutput<Flavor> output =
```

Comment:
Since we are in a source file, add `using bb::stdlib::recursion::honk`--will be surprsied if this leads to namespace conflicts.